CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.10)
 project(inputstream.adaptive)
+option(BUILD_TESTING "Build the testing tree." ON)
```

Comment:
we use ENABLE_TESTING in kodi

Reply:
Is that a problem?

Perhaps adding the condition `CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME` would be the right behavior?

Reply:
it should be consistent. I assume that other addons will sooner or later get tests as well.
Once more addons have tests, this should be added to the addon helper scripts in kodi.